### ISSUE TITLE
LSODA build requires administrator rights on Windows?

### ISSUE THREAD

**Issue by jibaneza** [rveltz/LSODA.jl#62]

**Comment by ChrisRackauckas** [comment]
```julia 
(v1.3) pkg> add LSODA
  Updating registry at `C:\Users\accou\.julia\registries\General`
  Updating git-repo `https://github.com/JuliaRegistries/General.git`
 Resolving package versions...
 Installed CodeTools ─ v0.6.5
 Installed QuadGK ──── v2.2.0
  Updating `C:\Users\accou\.julia\environments\v1.3\Project.toml`
  [7f56f5a3] + LSODA v0.6.1
  Updating `C:\Users\accou\.julia\environments\v1.3\Manifest.toml`
  [53a63b46] ↑ CodeTools v0.6.4 ⇒ v0.6.5
  [7f56f5a3] + LSODA v0.6.1
  [1fd47b50] ↑ QuadGK v2.1.1 ⇒ v2.2.0

julia> using LSODA
[ Info: Precompiling LSODA [7f56f5a3-f504-529b-bc02-0b1fe5e64312]
```

and tests pass, so I can't reproduce. Can you give more details on what issue you're seeing?

**Comment by jibaneza** [comment]
Further testing, I see that the error is related to administrator rights. For some reason it needs administrator rights for the build step. When using a non-administrator user I get:

`Error: Error building `LSODA`:
│ [ Info: Attempting to create directory C:\Users\xyz\.julia\packages\LSODA\uaBH4\deps\downloads
│ [ Info: Downloading file https://dl.bintray.com/sdwfrost/generic/liblsoda.7z
│ [ Info: Done downloading file https://dl.bintray.com/sdwfrost/generic/liblsoda.7z
│ [ Info: Attempting to create directory C:\Users\xyz\.julia\packages\LSODA\uaBH4
│ [ Info: Directory C:\Users\xyz\.julia\packages\LSODA\uaBH4 already exists
│ ERROR: LoadError: IOError: could not spawn `'C:\Users\xyz\AppData\Local\Julia-1.3.0\bin\7z.exe' x 'C:\Users\xyz\.julia\packages\LSODA\uaBH4\deps\downloads\liblsoda.7z' -y '-oC:\Users\xyz\.julia\packages\LSODA\uaBH4\deps'`: no such file or directory (ENOENT)
│ Stacktrace:
│  [1] _spawn_primitive(::String, ::Cmd, ::Array{Any,1}) at .\process.jl:99
│  [2] setup_stdios(::Base.var"#554#555"{Cmd}, ::Array{Any,1}) at .\process.jl:112
│  [3] _spawn at .\process.jl:111 [inlined]
│  [4] #run#565(::Bool, ::typeof(run), ::Cmd) at .\process.jl:439
│  [5] run(::Cmd) at .\process.jl:438
│  [6] run(::BinDeps.PathRule) at C:\Users\xyz\.julia\packages\BinDeps\ZEval\src\BinDeps.jl:503
│  [7] run(::BinDeps.SynchronousStepCollection) at C:\Users\xyz\.julia\packages\BinDeps\ZEval\src\BinDeps.jl:521
│  [8] satisfy!(::BinDeps.LibraryDependency, ::Array{DataType,1}) at C:\Users\xyz\.julia\packages\BinDeps\ZEval\src\dependencies.jl:944
│  [9] satisfy!(::BinDeps.LibraryDependency) at C:\Users\xyz\.julia\packages\BinDeps\ZEval\src\dependencies.jl:922
│  [10] top-level scope at C:\Users\xyz\.julia\packages\BinDeps\ZEval\src\dependencies.jl:977
│  [11] include at .\boot.jl:328 [inlined]
│  [12] include_relative(::Module, ::String) at .\loading.jl:1105
│  [13] include(::Module, ::String) at .\Base.jl:31
│  [14] include(::String) at .\client.jl:424
│  [15] top-level scope at none:5
│ in expression starting at C:\Users\xyz\.julia\packages\LSODA\uaBH4\deps\build.jl:39
└ @ Pkg.Operations D:\buildbot\worker\package_win64\build\usr\share\julia\stdlib\v1.3\Pkg\src\backwards_compatible_isolation.jl:649`

**Comment by ChrisRackauckas** [comment]
The real fix to this is to move to the newer binary builder systems.

**Comment by ChrisRackauckas** [comment]
This is fixed in v0.7 with the change to binary builder JLLs.

**Comment by ChrisRackauckas** [comment]
@sdwfrost you might be curious to check out https://github.com/JuliaBinaryWrappers/LSODA_jll.jl which has all of the prebuilt binaries in the Yggdrasil system:

https://github.com/JuliaBinaryWrappers/LSODA_jll.jl/releases/tag/LSODA-v0.1.1%2B0
